README.md

```diff
@@ -1 +1,29 @@
-# ecommerce-team4
\ No newline at end of file
+# Ecommerce Project - Team4
+
+This website includes two parts one for customers and one for sellers , the site shows 
+all the avilable items the customer can search items by name and can filter them by
+price or category, he can add items to cart , delete them from cart and see total price 
+of items in cart.
+the seller can add items edit them or delete them, and can search in his products.
+
+# Design link:
+
+# Github pages link:
+https://gsg-k1-fc.github.io/ecommerce-team4/.
+
+# File structure:
+files include the following files,
+- index.html file which containe html code.
+- script.js file which contain all javascript code.
+- style.css file which contain css style code.
+- assests folder which contain the images files.
+
+# How to run the project:
+to run the project you need to download this reapo and opening the index.html file in the browser.
```

Comment:
do it like that: 

- clone the repo into your local machine 
`git clone git@github.com:gsg-K1-FC/ecommerce-team4.git`

- Open the HTML on your browser

Reply:
ok